src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1332,6 +1332,36 @@ def getProcessingString(self):
         """
         return getattr(self.data.parameters, 'processingString', None)
 
+    def getCMSSWVersionsWithMergeTask(self):
+        """
+        _getCMSSWVersionsWithMergeTask_
+
+        Get the all the cmssw versions for task pluch merge task cmssw version.
```

Comment:
Maybe just update this string saying it gets all cmssw versions for this task and its first generation children.